components/org.wso2.ppaas.manager.console/console/controllers/menu/menu_generator.jag

```diff
@@ -36,14 +36,27 @@ function menuGenerator(context, jsonobject, permissions) {
                     hasPermission = intersection(jsonPermission.permissionPaths, permissions);
 
             if (hasPermission.length > 0) {
-                if(jsonPermission.linkexternal){
-                    link = jsonPermission.link;
-                    target = "_blank";
-                }else{
-                    link = context + '/' + jsonPermission.link;
+                if (jsonPermission.title == "Monitoring") {
+                    if (jsonPermission.linkexternal) {
+                        target = "_blank";
+                    }
+                    if (application.get("dasStatsPublisherEnabled") == true) {
+                        link = application.get("monitoringDashboardUrl");
+                        menu_html += '<li><a href="' + link + '" target="' + target + '">';
+                    } else {
+                        link = jsonPermission.link;
+                        menu_html += '<li><a href="' + link + '" target="' + target + '" class="disabled">';
+                    }
+                } else {
+                    if (jsonPermission.linkexternal) {
+                        link = jsonPermission.link;
+                        target = "_blank";
+                    } else {
+                        link = context + '/' + jsonPermission.link;
+                    }
+                    menu_html += '<li><a href="' + link + '" target="' + target + '">';
                 }
```

Comment:
I have already done the formatting. Could you please point out at which lines code is not formatted.